README.md

```diff
@@ -102,9 +122,10 @@ The following properties are already defined in the default configurations:
 
 ### Prefixes
 
-It's possible to use prefixes for specific paths in the property values.
+It's possible to use prefixes in the property values which will be translated to specific paths or environment variable values.
```

Comment:
I am not sure if this is fully understandable. Retry:

> It is possible to use prefixes in the property values of the configuration. These prefixes will be translated to specific paths or environment variable values.

I guess the main idea is better readability right? Maybe one could add that too